srcpkgs/recutils/template

```diff
@@ -1,18 +1,31 @@
 # Template file for 'recutils'
 pkgname=recutils
 version=1.8
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--with-bash-headers --disable-rpath"
-hostmakedepends="pkg-config"
+# TODO remove automake in 1.9; only needed for copy-file
+hostmakedepends="automake pkg-config"
 makedepends="acl-devel libgcrypt-devel libuuid-devel libcurl-devel"
 short_desc="Utilities to deal with recfiles"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://www.gnu.org/software/recutils/"
 distfiles="${GNU_SITE}/recutils/recutils-${version}.tar.gz"
 checksum=df8eae69593fdba53e264cbf4b2307dfb82120c09b6fab23e2dad51a89a5b193
+patch_args=-Np1
 
+pre_configure() {
+	if test x"$version.$revision" != x1.8.2 ; then
+		echo hopefully you can remove the automake dependency
+		echo and the pre_configure kludge
+		exit 1
+	fi
```

Comment:
I don't this is necessary, you should trust the comments...

Reply:
Ok, it was just en enforceable reminder.
Btw, I just realized something - I did the "touch man/*.1" kludge because `xbps-query -Rvs help2man` showed nothing,
but I'm on my debian computer, and even `xbps-query -Rvs vim` shows nothing.
That being said, the manpages *should* stay unchanged, as this is just a bugfix with zero functionality change.